### DISCUSSION TITLE
A way to customize the toast / snackbar notification?

### DISCUSSION THREAD

**Discussion by mrgrlscz** [home-assistant/frontend#20427] [Theming]
I'm struggling to customize the toast / snackbar in HA frontend.
Right now i'm only able te modify the border-radius, by stting up "mdc-shape-small: 8px" in my theme_name.yaml
Maybe it's not the right way.

My goal is to set colors, background AND text. And maybe modifyin" the "justify-content" and "margin" values.
It was working well before 2024.4.0 (only for color), by setting up "paper-toast-background-color: rgb(x,x,x)" in the theme file.

**Comment by Misiu** [comment]
paper toast was replaced with mwc-snackbar (https://github.com/home-assistant/frontend/pull/19579), so that's probably the reason.
Not sure what CSS variables should be used after the change or if this is even possible (ref: https://github.com/material-components/material-web/issues/2162#issuecomment-1001703348)

**Comment by mrgrlscz** [comment]
I knew it was replaced, and thanks for the second link.
My entire dashboard is built around the 'old' toast specs (especially height)..... 😭
